site/guide/_manage-roles.qmd

```diff
@@ -0,0 +1,21 @@
+1. Under User Permissions, select **Roles**.
```

Comment:
I would move the list you have under 5. here. You're also missing some information about how you can see the list of users currently assigned to this role which is actually the core part of the article, since this is the one about roles and not permissions! (Even though roles HAVE permissions...)